cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,23 @@
+const USER_EMAIL = "ronswanson@noroff.no";
+const USER_PASSWORD = "bacon1337";
+
+describe("LogInAndOut", () => {
+  beforeEach(() => {
+    cy.wait(500);
+    cy.visit("/");
+  });
+  it("Can log in with valid credentials then log out with the logout button", () => {
+    cy
+      .get("#registerModal button")
+      .contains("Login")
+      .should("be.visible")
+      .wait(500)
+      .click()
+      .wait(500),
+      cy.get(`input#loginEmail[name="email"]`).type(USER_EMAIL);
+    cy.get(`input#loginPassword[name="password"]`).type(USER_PASSWORD);
+    cy.get(`button[type="submit"]`).contains("Login").click();
+    cy.wait(5000);
```

Comment:
http://i.imgur.com/CWfyFbB.jpg

This test is very "witty" in the sense that it uses a very simple assumption (you can't log out if you aren't logged in) to measure auth state. While this is concise, the lack of explicitness gives me concern.

I would add a step here and use that to check tokens in local storage:

```js
cy.get(`button[type="submit"]`).contains("Login").click().should(() => {
  expect(localStorage.token).to.be.truthy
})
```

and later you can do the opposite to check that the token was removed

Reply:
(please bear in mind I'm writing pseudocode so my syntax might not be 100%)

Reply:
Relevant: https://content.noroff.dev/workflow/intro-to-testing.html#stating-the-obvious

Reply:
Changes has been made. It was not quite as simple as your comment suggests, but it did not take too long to resolve it.